dot_net/CSharpDotNetSamples/CaptureJson.aspx.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Web;
+using System.Web.UI;
+
+namespace AspNetClientEncryptionExample

```

Comment:
It was a name of the project earlier – AspNetClientEncryptionExample. I was thinking that it should have changed it
But let me look into it and get back to you. In C# all the classes will be a part of the namespace. And this namespace name will be ideally the name of the project but you can give any other name.

Thanks,

RUPA MODHA  DEVELOPER SUPPORT ENGINEER
PayTrace The Secure Advantage

T 509 228 8033<applewebdata://A56E5FD9-6AB4-451E-82A4-B5A8816D73C2/tel/%20360%20635%206291> · F 360 326 1985<applewebdata://A56E5FD9-6AB4-451E-82A4-B5A8816D73C2/tel/%20360%20326%201985>
Rmodha@PayTrace.com<mailto:Joel@PayTrace.com>· https://paytrace.com<https://paytrace.com/>
Twitter @PayTrace<https://twitter.com/PayTrace> · LinkedIn PayTrace<http://www.linkedin.com/company/paytrace>





From: Eric Kimbrel <notifications@github.com<mailto:notifications@github.com>>
Reply-To: PayTrace/api-demo <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, March 7, 2016 at 3:01 PM
To: PayTrace/api-demo <api-demo@noreply.github.com<mailto:api-demo@noreply.github.com>>
Subject: Re: [api-demo] Feature/c sharp dot net samples (#5)


In dot_net/CSharpDotNetSamples/CaptureJson.aspx.cs<https://github.com/PayTrace/api-demo/pull/5#discussion_r55288894>:

> @@ -0,0 +1,178 @@
> +using System;
> +using System.Web;
> +using System.Web.UI;
> +
> +namespace AspNetClientEncryptionExample


I don't know a lot about C# namespaces but it looks like you are using the AspNetClientEncryptionExample namespace in a lot of places like this that aren't really related to ClientEncryption. If the same namespace needs to be used everywhere for things to work you might want to consider changing the namespace to something more generic to avoid confusion about it.

—
Reply to this email directly or view it on GitHub<https://github.com/PayTrace/api-demo/pull/5/files#r55288894>.